packages/sdk-ui-ts/src/utils/helpers.ts

```diff
@@ -15,3 +16,28 @@ export const getTokenLogoWithVendorPathPrefix = (image: string) => {
 
   return path.join('/', 'vendor', '@injectivelabs', 'token-metadata', image)
 }
+
+export const getTokenLogoFromTokenType = (token: Token) => {
```

Comment:
Let's move the non-hardcoded logo to the UI, which will give us some room for flexibility. Example: CW20 logo could be a byte64 string, and we might want to do some kind of customization to render them the the user.

```
case tokenType === TokenType.Cw20 && logo:
      return logo
case isIbcOrEthTokenWithVendorLogo:
      return getTokenLogoWithVendorPathPrefix(logo)
```

Reply:
@ThomasRalee I removed the non-hardcoded logo back to the UI level